pkg/memo/memo_test.go

```diff
@@ -0,0 +1,264 @@
+package memo_test
```

Comment:
```suggestion
package memo
```

Reply:
This package should remain "pure" for testing.

Reply:
Can you clarify a bit on `not use sample.*?` The tests need to sample data for the encoder/decoder to work on or compare with.

Reply:
I removed the `sample` package from dependency https://github.com/zeta-chain/node/pull/2987/commits/4f5b1a1f1bc078359bfb411a73ad948f999e3cc5